ui/perfherder/SortButton.jsx

```diff
@@ -8,17 +8,17 @@ import {
   faSortUp,
 } from '@fortawesome/free-solid-svg-icons';
 
-import { compareTableSort } from '../constants';
+import { tableSort } from './sortHelpers';
```

Comment:
I think it's better to move this component in a `shared` directory at the perfherder level, not treeherder since we reuse this component only in perfherder.